[RM Ch02] GSMA fedback addressed (#1752)
cntt-n/CNTT: doc/ref_model/chapters/chapter02.md

This means, operators will be able to host their Telco workloads (VNFs/CNFs) with different traffic types, behaviour and from any vendor on a unified consistent cloud infrastructure.

> Additionally, a well defined cloud infrastructure is also needed for other type of workloads such as IT, Machine Learning, Artificial Intelligence, etc.

Reviewer: @walterkozlowski Since the sentence states, "such as", may I suggest the following:
```suggestion
Additionally, a well defined cloud infrastructure is also needed for other type of workloads such as IT, Machine Learning, and Artificial Intelligence.
```

> This chapter tries to analyse various workload types used telco and examine their requirements. The chapter will also highlight some of the cloud infrastructure parameters needed to achieve the desired performance expected by various workloads.

Reviewer: ```suggestion
This chapter analyses various telco workloads and their requirements, and recommends certain cloud infrastructure parameters needed to specify the desired performance expected by these workloads.
```

  - Value-Added-Services
- **By technology:** 2G, 3G, 4G, 5G, IMS, FTTx, Wi-Fi...

> Below is a list of Network Functions that covers almost _**95%**_ of the Telco workloads (and the most likely to be virtualised). They are gathered by network segment and function type.

Reviewer: ```suggestion
The list of, most likely to be virtualised,  Network Functions below, covering almost _**95%**_ of the Telco workloads, is organised by network segment and function type.
```

<p align="center"><img src="../figures/ch02_infra_profiles.PNG" alt="infra_profiles" title="Infrastructure Profiles" width="100%"/></p>
<p align="center"><b>Figure 2-1:</b> Infrastructure profiles proposed based on VNFs categorisation.</p>

> In **Chapter 4** later in the document, these infrastructure profiles will be offered to workloads: **B (Basic)** and **N (Network intensive)** <!--, and **C (Compute intensive)** --> respectively.

Reviewer: What is the "respectively" for? Not sure what is means "offered" to workloads? In Chapter 4 these profiles are further specified.
```suggestion
In **[Chapter 4](./chapter04.md)** these **B (Basic)** and **N (Network intensive)** <!--, and **C (Compute intensive)** --> infrastructure profiles will be defined in greater detail for use by workloads: .
```
